ui/src/components/request/PolicyRequestsReview.js

```diff
@@ -38,14 +33,121 @@ class PolicyRequestReview extends Component {
     };
     this.reloadDataFromBackend = this.reloadDataFromBackend.bind(this);
     this.updatePolicyDocument = this.updatePolicyDocument.bind(this);
-    this.updateRequestStatus = updateRequestStatus.bind(this);
-    this.sendProposedPolicy = sendProposedPolicy.bind(this);
   }
 
   componentDidMount() {
     this.reloadDataFromBackend();
   }
 
+  async sendProposedPolicy(command) {
+    const { change, newStatement, requestID } = this.state;
```

Comment:
we need to move this review page to React Hooks soon. This code only works with binding to the specific components. This is a utility function not bound to this class instead of being used for the children components.